apps/website/src/pages/_global.astro

```diff
@@ -74,6 +74,15 @@ const { content = {} } = Astro.props;
         font-size: var(--type-3);
       }
 
+      :where(h2:not(.astro-4EWMQMN4):not(left-sidebar)) {
```

Comment:
1. why are you targeting this hashed class name? it's not meant to be used in source code as it can change at any time.
2. why are you adding this style to the global reset?

it will affect random parts of the website and break at any time
<img width="839" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/5baf997d-15f4-495a-bde7-98da1cae4cb1">

Reply:
the correct place to add this style is inside `.content-wrapper` in `_layout.astro`. you can add it after [line 108](https://github.com/iTwin/iTwinUI/blob/a4e00f411f3f3be5cfd75267ae06ebb1484fd670/apps/website/src/pages/docs/_layout.astro#L108) for example

Reply:
fixed